server/common.js

```diff
@@ -41,4 +41,23 @@ const userTypes = Object.freeze({
     ADMIN     : 0,
     STUDENT   : 1
 });
-exports.userTypes = userTypes;
\ No newline at end of file
+
+exports.userTypes = userTypes;
+
+const questionAttributes = Object.freeze({
```

Comment:
good point. lets put them as common, but we also need the other question types' attributes here. lets rename this to general question attributes, then we can add regular, multiple choice true and false etc... as their own question attributes

Reply:
That is right, we should have other data types in this as well under different key/values.